api/proto/external.proto

```diff
@@ -370,6 +370,10 @@ message MintConfigTxPrefix {
 
     /// The block index at which this transaction is no longer valid.
     uint64 tombstone_block = 4;
+
+    /// The maximal amount that can be minted by configurations specified in
+    /// this tx. This amount is shared amongst all configs.
+    uint64 total_mint_limit = 5;
```

Comment:
The nodes do not issue the minting, minting is issued by a client that submits it to a node. The client needs to have access to the private signing keys to properly sign the transaction, otherwise the nodes will reject it.
If the transaction asks for a mint that exceeds the limit it will be considered invalid by consensus and rejected.

Reply:
Cool, thanks for the explanation.